src/cliAction/startStopDelete/handler_cmdSwitch.go

```diff
@@ -0,0 +1,55 @@
+package startStopDelete
+
+import (
+	"context"
+
+	"github.com/zerops-io/zcli/src/constants"
+	"github.com/zerops-io/zcli/src/i18n"
+)
+
+type Method func(ctx context.Context, h *Handler, projectId string, serviceId string) (string, error)
```

Comment:
you can pass the function with it's receiver like this `h.ProjectStart` for example

Reply:
and then there is no need for the `Handler` in the function signature (which is then more general)